integration/integration_test.go

```diff
@@ -16,7 +16,7 @@ import (
 var kubebenchImg = flag.String("kubebenchImg", "aquasec/kube-bench:latest", "kube-bench image used as part of this test")
 var timeout = flag.Duration("timeout", 10*time.Minute, "Test Timeout")
 
-func TestRunWithKind(t *testing.T) {
+func TestCheckCIS13WithKind(t *testing.T) {
```

Comment:
I see the point of having separate test functions for the different clusters, so they can easily be run independently. But it seems like a lot of repeated code. The part that actually sets up the cluster and runs the test could be pulled into a separate function that each of the three test function calls

Reply:
Fixed.